awssqs/config/201-clusterrole.yaml

```diff
@@ -46,6 +46,19 @@ rules:
   - patch
   - delete
 
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
```

Comment:
Why does this use an explicit list, but the couchdb YAML and others use a reference?

Reply:
I followed the pattern in use in whatever file I was modifying, which follow varying conventions.